openfisca_france/model/prelevements_obligatoires/impot_revenu/reductions_impot.py

```diff
@@ -4121,10 +4251,12 @@ class rehab(Variable):
     def formula_2017_01_01(foyer_fiscal, period, parameters):
         '''
         Travaux de réhabilitation des résidences de tourisme
-        2017
+        (2017 -)
         '''
-        depenses_2017 = foyer_fiscal('f7xx', period)  # déjà plafonnées
+        f7xx = foyer_fiscal('f7xx', period)  # Comment coder le plafond roulant sur les 3 années ?
```

Comment:
On peut en discuter de cette histoire de plafond roulant

Reply:
TO DO marqué